src/osa/scripts/closer.py

```diff
@@ -484,6 +486,39 @@ def daily_datacheck(cmd: List[str]):
     log.info("Daily dl1 checks using longterm script.")
     log.debug(f"Executing {stringify(cmd)}")
 
+    if not options.simulate and not options.test and shutil.which("sbatch") is not None:
+        job = subprocess.run(cmd, check=True)
```

Comment:
One last thing, in order to sbatch to return the job id, you need to add `--parsable` argument after sbatch cmd in `daily_longterm_cmd` like:

```
"sbatch",
"--parsable",
"-D",
options.directory,
"-o",
...
```